### DISCUSSION TITLE
TCL file executed for open project but couldn't open GUI with this project

### DISCUSSION THREAD

**Discussion by Mendandukka** [polarfire-soc/polarfire-soc-documentation#259] [How to] [answered]
I was using and implemented several commands by using Libero SOC tool command reference guide. Finally i open project with executing configured tcl file. But i couldn't open GUI with already open project. I tried to use  os.system(/LIBERO DIR/bin/libero) for open GUI manually subprocessing with execute tcl file for open project. But in this scenario just GUI starting page is open.

I would appreciate if you could share your any advices or solutions.

**Comment by hughbreslin** [comment, marked as answer]
Hey @Mendandukka there is no way to open the GUI and pass a Tcl command through arguments unfortunately. You can either go the batch / Tcl flow where you run headlessly with no GUI or you can run the GUI flow where you run all commands via the GUI. You can open the GUI and manually pass a script (from "Project" -> "Execute Script") but that is as close as you can get.
